### DISCUSSION TITLE
Create attributs at each ERA5L cell

### DISCUSSION THREAD

**Discussion by jnthnroy** [kratzert/Caravan#29] [Q&A]
Hi!
Base on this repository, it is possible to generate attributs at each ERA5L cell in lieu of average bassin? Create a grid of attributs who match with ERA5L. May be it is already possible and I miss this option or you know a simple way to hack the code?
Thank you!

**Comment by kratzert** [comment]
Hi, sorry for the delayed reply. This is nothing that I specifically considered for the code. What you could do is to create a polygon layer with one polygon matching one ERA5-Land cell, and then running the code to get the attributes. But I am not 100% sure if everything works as expected though^^
